salesforce_api/kafka_salesforce_integration/kafka-salesforce-pricebook_update/sample.bal

```diff
@@ -0,0 +1,44 @@
+import ballerinax/kafka;
+import ballerinax/salesforce as sfdc;
+
+configurable string salesforceAccessToken = ?;
+configurable string salesforceBaseUrl = ?;
+public type ProductPrice readonly & record {|
+    string name;
+    float unitPrice;
+|};
+
+public type ProductPriceUpdate readonly & record {|
+    float UnitPrice;
+|};
+
+listener kafka:Listener orderListener = new (kafka:DEFAULT_URL, {
+    groupId: "order-group-id",
+    topics: "foobar"
+});
+
+const string PRICEBOOKID = "";
```

Comment:
What is purpose of this const which is always ""

Reply:
It is used to indicate the Pricebook ID of the desired pricebook. Will migrate it to Config.toml

Reply:
Updated in [bcf09ac](https://github.com/ballerina-guides/integration-samples/pull/31/commits/bcf09ac42e87543485854586312facacacd050cb)